plugin.video.gamegurumania/resources/__init__.py

```diff
@@ -0,0 +1 @@
+# Enables relative imports
```

Comment:
In python3 your init.py placeholder files shouldn't be needed

Reply:
Fixed